experimenter/tests/integration/nimbus/utils/filter_expression.js

```diff
@@ -26,22 +24,34 @@ async function remoteSettings(targetingString, recipe) {
         const evalResult = await targetingContext.evalWithDefault(targetingString);
         return evalResult !== undefined;
     } catch (err) {
-        return null;
+        throw err;
     }
 }
 
-/*
-Arguments contains 3 items.
-arguments[0] - the JEXL targeting string
-arguments[1] - the experiment recipe
-arguments[3] - the callback from selenium
-*/
-const [targetingString, recipe, callback] = arguments;
+async function main() {
+    // Initialize once
+    await TelemetryEnvironment.onInitialized();
+    await ExperimentAPI.ready();
+
+    const [testsJson, callback] = arguments;
```

Comment:
You have to set this outside the `main` function because `arguments` is a special global that refers to the arguments passed to the current function. ie so in this function it will be `[]`